xsuite/src/world/signer.ts

```diff
@@ -97,6 +106,10 @@ export class Keystore {
     return UserWallet.decryptMnemonic(this.data, this.password).getWords();
   }
 
+  getSecretKey() {
+    return UserWallet.decryptSecretKey(this.data, this.password);
```

Comment:
What's the return type of `decryptSecretKey`? If it is a type specific to the MultiversX SDK, let's return `.hex()` instead. I guess `.hex()` doesn't return a type to the MultiversX SDK?

Reply:
It is `UserSecretKey`. At first I returned `.hex()` but decided to return `UserSecretKey` instead. See [here](https://github.com/arda-org/xSuite/pull/70#discussion_r1436510911).

I did that because `UserWallet.fromSecretKey()` used in #71 requires its `secretKey` option to be a `UserSecretKey`.

But I found that `UserSecretKey` could also be created using `UserSecretKey.fromString()`. 
Thus, I now return `.hex()` as you wanted, and made the relevant changes in the code to support it.

Reply:
What is the return type of `.hex()`? Ideally, we would want a type that is not specific to MultiversX SDK.

Reply:
It is string, so it is good :+1: